### PR TITLE
Add numerical drop-down button in Content Editor

### PR DESCRIPTION
**Pull request by jessouyangbraven**
### Description

Task: https://app.asana.com/0/1174274412967132/1198222893411568

This is a follow up to #451 that allows a Designer to click the **1-10 Dropdown** button in the **Design** tab fo the Content Editor and insert the selector.

### Motivation and Context

This is easier than copy-pasting the HTML into an un-indented blob of HTML.

### Screenshots (if appropriate):

- In Content Editor, click the **1-10 Dropdown** button to insert the selector:
Screen Shot 2020-11-12 at 5 03 22 PM

- The `name` attribute (and `data-bz-retained` in case we use this in projects and not just surveys) is automatically set by the `Command` and uses a new ID from `UniqueID`:
Screen Shot 2020-11-12 at 5 03 34 PM

- **Save** this as a _Survey_ and publish to Canvas using Course Management page:
Screen Shot 2020-11-12 at 5 04 29 PM

- As a Fellow, choose a value and **Submit**:
Screen Shot 2020-11-12 at 5 05 05 PM

- We don't have a UI to see impact survey responses, so use the Rails console to check that `SurveySubmissionAnswer` was created:
Screen Shot 2020-11-12 at 5 05 33 PM

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/javascript/ckeditor/insertnumericalselectorcommand.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import Command from '@ckeditor/ckeditor5-core/src/command';
+import UniqueId from './uniqueid';
+import { ELEMENT_ID_PREFIX } from './uniqueid';
```
Comment (rshipp) [inline app/javascript/ckeditor/insertnumericalselectorcommand.js:3]: you can remove this import

File: app/javascript/components/ContentEditor.js [hunk 2]
```diff
@@ -89,7 +89,6 @@ BalloonEditor.builtinPlugins = [
     Font,
     SimpleUploadAdapter,
     HorizontalLine,
-
```
Comment (rshipp) [inline app/javascript/components/ContentEditor.js:92]: nit, but can you put this whitespace back? it's just a visual separation between the official cke plugins and our custom ones.

**Review by sadleb — COMMENTED** [review]
Defer to Ryan on this PR too.

**Review by sadleb — APPROVED** [review]
Looks like Ryan took a pass at reviewing this so even though he didn’t explicitly approve, of you’ve tested and this works, I’m comfortable merging before the demo without waiting for an explicit approval from him

**Pull request merged**